src/main/java/org/galaxyproject/gxformat2/CytoscapeDAG.java

```diff
@@ -0,0 +1,162 @@
+package org.galaxyproject.gxformat2;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/**
+ * @author gluu
```

Comment:
Link to documentation, presumably Cytoscape has a schema or at least examples of this data model